src/BSON/Timestamp.c

```diff
@@ -285,10 +337,12 @@ HashTable *php_phongo_timestamp_get_properties(zval *object TSRMLS_DC) /* {{{ */
 
 		MAKE_STD_ZVAL(increment);
 		ZVAL_LONG(increment, intern->increment);
+		convert_to_string(increment);
```

Comment:
To elaborate: since timestamp and increment are `uint32_t`, they could exceed the range of signed 32-bit ints. ZVAL_LONG is being used to stuff this unsigned value into a 32-bit integer (within the zval struct), which can obviously hold the value but it might be interpreted as a negative value later on. My concern is that `convert_to_string()` might render `4294967295` as `"-2147483648"`.